product/models.py

```diff
@@ -0,0 +1,95 @@
+from django.db import models
+
+# 상품
+class Product(models.Model):
+    category        = models.ForeignKey('category', on_delete=models.CASCADE)
+    name            = models.CharField(max_length=50)
+    price           = models.DecimalField(max_digits=8, decimal_places=2)
+    stock           = models.IntegerField()
+    sale            = models.IntegerField()
+    user            = models.ManyToManyField('user.user', through='productlike',related_name='product_like')
+    create_at       = models.DateTimeField(auto_now_add=True)
+    update_at       = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return f'{self.name}'
+    
+    class Meta:
+        db_table = 'products'
+
+# 카테고리
+class Category(models.Model):
+    name            = models.CharField(max_length=20)
+
+    def __str__(self):
+        return f'{self.name}'
+
+    class Meta:
+        db_table = 'categories'
+
+# 상품 이미지
+class ProductImage(models.Model):
+    product         = models.ForeignKey('product', on_delete=models.CASCADE)
+    image_url       = models.CharField(max_length=4000)
```

Comment:
```suggestion
    image_url       = models.URLField(max_length=4000)
```
CharField와 같지만, 장고에는 URLField라는 것도 있으므로 써봅시다. 가독성 측면의 권장입니다.

Reply:
오 감사합니다!